File: test/fixtures/java-pkg/com/example/Calculator.java
package com.example;

import java.util.List;
import java.util.Map;
import com.example.Helper;

/**
 * Adds and manages numbers.
 */
public class Calculator {
    public int add(int a, int b) {
        return a + b;
    }

    private int secret() {
        return 0;
    }
}

public interface Shape {
    double area();
}

public enum Color { RED, GREEN, BLUE }

File: test/fixtures/kotlin-pkg/com/example/Calculator.kt
package com.example

import java.util.List
import kotlin.math.PI

/**
 * Adds two ints.
 */
fun add(a: Int, b: Int): Int {
    return a + b
}

private fun secret(): Int {
    return 0
}

class Point(val x: Int, val y: Int) {
    fun sum(): Int {
        return x + y
    }

    private fun hidden(): Int {
        return 0
    }
}

interface Shape {
    fun area(): Double
}

enum class Color { RED, GREEN, BLUE }
